tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -369,6 +369,28 @@ public void EnsureAllServiceBusManifestTypesHaveVersion0Suffix()
         Assert.Equal("azure.servicebus.v0", servicebus.GetProperty("type").GetString());
     }
 
+    [Fact]
+    public void EnsureAllAzureDatabaseManifestTypesHaveVersion0Suffix()
```

Comment:
This change seems unrelated to the overall PR.

Reply:
It isn't. Thought I could sneak it in as it's just a missing test. Do you care about it being a different PR/commit?

Reply:
It's fine to add it here. Just making sure it is intentional.